tests/models/test_dag.py

```diff
@@ -56,6 +56,7 @@
 from airflow.models.param import DagParam, Param, ParamsDict
 from airflow.models.serialized_dag import SerializedDagModel
 from airflow.operators.bash import BashOperator
+from airflow.operators.dummy import DummyOperator
```

Comment:
We have `EmptyOperator`

Reply:
Thanks @eladkal . Fixed and rebased with latest main branch.